tests/e2e/kubetest2-kops/deployer/deployer.go

```diff
@@ -91,8 +91,11 @@ type deployer struct {
 
 	// boskos struct field will be non-nil when the deployer is
 	// using boskos to acquire a GCP project
-	boskos *client.Client
-
+	boskos                  *client.Client
+	BoskosLocation          string        `flag:"~boskos-location" desc:"If set, manually specifies the location of the Boskos server."`
```

Comment:
Nit: can we start this with a hyphen?  Other flags in this struct do not.

Reply:
I copied that kubetest2-gce deployer, getting rid of it now

Reply:
Fixed